sympy/functions/elementary/piecewise.py

```diff
@@ -663,9 +663,9 @@ def handler(ipw):
         return sum
 
     def _intervals(self, sym):
-        """Return a list of tuples, (a, b, e, i), where a and b
+        """Return a list of unique tuples, (a, b, e, i), where a and b
         are the lower and upper bounds in which the expression e
-        of argument i in self is defined.
+        of argument i in self is defined and a != b.
```

Comment:
Should this be "a < b"?

Reply:
That will be true if a and b are numbers. When symbolic they may become equal upon evaluation. For example:

```
>>> Piecewise((1,And(x<z,x>y,x>0)),(2,True))._intervals(x)
[(Max(0, y), Max(0, y, z), 1, 0), (-oo, oo, 2, 1)]
>>> [Tuple(*i).subs({y: 2, z: 1}) for i in _]
[(2, 2, 1, 0), (-oo, oo, 2, 1)]
```

I'll try rephrase this.